spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseDeleteOrphanFilesSparkAction.java

```diff
@@ -263,7 +276,15 @@ private static void listDirRecursively(
       int maxDirectSubDirs = Integer.MAX_VALUE;
 
       dirs.forEachRemaining(dir -> {
-        listDirRecursively(dir, predicate, conf.value().value(), maxDepth, maxDirectSubDirs, subDirs, files);
+        listDirRecursively(
+            dir,
+            predicate,
+            conf.value().value(),
+            maxDepth,
+            maxDirectSubDirs,
+            subDirs,
+            files,
+            includeHiddenPaths);
```

Comment:
Can you revert the formatting change here? We just need to add `includeHiddenPaths`.